prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1286,6 +1286,14 @@ int rxr_ep_init(struct rxr_ep *ep)
 	if (ret)
 		goto err_free_rx_entry_pool;
 
+	ret = ofi_bufpool_create(&ep->rx_atomrsp_pool,
+				 ep->mtu_size,
+				 RXR_BUF_POOL_ALIGNMENT,
+				 RXR_MAX_RX_QUEUE_SIZE,
+				 ep->rx_size, 0);
```

Comment:
Can the initial size smaller? I will be surprised if we will use all 8k of these in normal use.

Reply:
Sure, make sense. I changed it to 1024 in new revision.